datastructures/dynamic-array/dynamicarray_test.go

```diff
@@ -0,0 +1,305 @@
+package dynamic_array
```

Comment:
if you do the above step then it would be better for this to be changed to `package dynamicarray` as well